### ISSUE TITLE
Adding/creating AutoEq (headphone) presets for FxSound

### ISSUE THREAD

**Issue by doolhoofd** [fxsound2/fxsound-app#22]
Multiple users have requested the addition/creation of AutoEq presets to be used in combination with FxSound.
AutoEq presets are automatically generated Equalizer settings tailor-made for various different headphone models, aimed mainly at neutralizing their respective frequency response output.
This is a link to AutoEq's page on GitHub:
https://github.com/jaakkopasanen/AutoEq/wiki
This is AutoEq's main website: 
https://autoeq.app

**Comment by Tarek-Hasan** [comment]
Thanks,
I was going to open an issue for it but found you already opened one.

**Comment by Bumbleboss** [comment]
Any news on this?

**Comment by doolhoofd** [comment]
> Any news on this?

No, and also not at the top of the priority list at the moment; sorry.

**Comment by realgooseman** [comment]
The AutoEQ database is richer, it would be too bad to not add a way to convert existing presets on AutoEQ to be easily importable on FX-Sound in the future.

I also created an issue on the AutoEQ repo for considering adding FX-Sound as an option to export EQ presets to.
Here is the issue, upvote it so it gains visibility: https://github.com/jaakkopasanen/AutoEq/issues/814

**Comment by Tarek-Hasan** [comment]
I would prefer if AutoEQ is integrated into the FxSound interface like [Peace Equalizer](https://sourceforge.net/projects/peace-equalizer-apo-extension/) or [AQUA](https://github.com/h39s/AQUA). Alternatively, you can include [Bard](https://sourceforge.net/projects/bard-browser-autoeq/); a AutoEQ manager with FxSound if you want less hassle.

**Comment by rayman89** [comment]
Is been almost 2 years. Can we get it please?
